storybook/mocks/globalUtils.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <QJsonArray>
+#include <QJsonDocument>
+#include <QJsonObject>
+
+class MockGlobalUtils : public QObject
+{
+    Q_OBJECT
+
+public:
```

Comment:
```suggestion
public slots:
```
and you won't need to repeat `Q_INVOKABLE` everywhere